src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -182,6 +182,20 @@ const ContractDetails = ({
                             error={!!flatErrors['fundingSource.source']}
                             id="IntakeForm-FundingSource"
                             name="fundingSource.source"
+                            // manual onChange to catch case where user selects 'Unknown' funding source
+                            // TODO: I feel like there should be a better option for this use case
+                            //       but I could not find anything cleaner then this solution
+                            onChange={(e: any) => {
+                              formikProps.handleChange(e);
+
+                              // If funding source is 'Unknown' set funding number to 'N/A' (funding number
+                              // field is disabled in this case), for any other funding source it is set
+                              // to empty string and the user can enter funding number normally
+                              setFieldValue(
+                                'fundingSource.fundingNumber',
+                                e.target.value === 'Unknown' ? 'N/A' : ''
```

Comment:
Why should this make the funding number `N/A`? It should just be empty if there's no funding number. I don't think we need to change the default behavior of the `onChange`.

Reply:
☝️ I take part of that back. You need to reset the funding number if the funding source changes to unknown.

Reply:
I liked just making the funding number value N/A rather then catching the empty string in the display and translating to N/A. Im happy to change it but that was my thinking.

Reply:
IMO, field values should be empty unless we need to store that it's `N/A`. Similarly, when funding source is marked as `No`, the dropdown `fundingSource` and `fundingNumber` are both `''` and not `N/A`.

Reply:
I'd display nothing for funding number. `Yes, Unknown`. I could even make an argument to only display `Yes`. It's not clear what "Unknown" means without any context, even less clear if we displayed `Yes, Unknown, N/A`.

Maybe something along the lines of the follow would be better:
```
if (source === 'Unknown') {
  return 'Yes, funding source and funding number are unknown'
}
```

Reply:
Ok I like it, I'll mess around with it. I realized I had some other work I wanted to do with this involving i18n so I'll bundle that in there. Thanks